projects/objects/balls/protos/FifaSoccerBall.proto

```diff
@@ -26,7 +26,7 @@ PROTO FifaSoccerBall [
     children [
       Transform {
         scale %{=scale}% %{=scale}% %{=scale}%
-        translation 0 %{=radius}% 0
+        translation 0 0 0
```

Comment:
Usually the PROTO nodes we provide are by default added above the ground.
With this change the ball will be half below the ground.
Is this really wanted? Because version seems better and this is the only ball from the `objects/balls` section that is not located above the default ground position.

Reply:
Yes, this is wanted.
It's because the previous version was not working in ENU coordinate system and it seems simpler to set the center at the center of ball.

Reply:
Ok. However this is not at all consistent with the PROTO we usual distribute with Webots and it is quite bad to have to set manually the translation value to make it above the ground (even because the radius it is not known).

I think we should find a better solution to the ENU coordinate system problem and restore the initial position 0 0 0 above the ground. This can also be done in a different PR.